### ISSUE TITLE
combine_conv_bn function should disable gradient，otherwise will get a runtime error

### ISSUE THREAD

**Issue by ghost** [ShiqiYu/SimpleCNNbyCPP#3]
![image](https://user-images.githubusercontent.com/24853152/117566220-f5a38380-b0e7-11eb-8119-2dd27af1e243.png)
This error can be easily fixed using @torch.no_grad()

**Comment by fengyuentau** [comment]
Can you provide more details on how you run the code, such as the version of torch and the command you run?

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: zhanghj2020 ***@***.***>
发送时间: Sunday, May 9, 2021 5:01:12 PM
收件人: ShiqiYu/SimpleCNNbyCPP ***@***.***>
抄送: Subscribed ***@***.***>
主题: [ShiqiYu/SimpleCNNbyCPP] combine_conv_bn function should disable gradient，otherwise will get a runtime error (#3)


[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F24853152%2F117566220-f5a38380-b0e7-11eb-8119-2dd27af1e243.png&data=04%7C01%7C%7C86ef88adced94af0448a08d912c8ff1a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637561476746552697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=X%2BNUZQkReQ4QAiXZBx9OjKF4xTpt5JEos5Fpj4IMAF8%3D&reserved=0>
This error can be easily fixed using @torch.no_grad()

―

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/24853152/117650265-14c31380-b1c3-11eb-95d8-c297091eae69.png)

**Comment by fengyuentau** [comment]
Fixed.
